public/speech-loader.html

```diff
@@ -16,11 +16,13 @@
 
       if (timeout <= now) {
         console.log('timeout exceeded waiting for voices, continuing anyway')
-        setTimeout(goToApp, oneSecond)
+        setTimeout(() => goToApp('reader=off'), oneSecond)
         return true
       } else {
         const seconds = Math.floor((timeout - now) / oneSecond)
-        console.log(`timeout not yet reached, still waiting for voices for ${seconds}s`)
+        console.log(
```

Comment:
I'm okay leaving these console messages in, just want to make sure you meant to.

Reply:
Yes, figured they'd be useful on this screen.